### ISSUE TITLE
Added new comment should be visible without needing scrolling

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/rtMedia#1464]
1.  Open media in a lightbox.
2.  Enter a comment for media in lightbox only.
3.  Observe that after a few comments, newly added comments will not be visible, need to scroll the comment section to view new comments added.

Refer Visual: https://drive.google.com/open?id=1aa0ELPJBTIjo8M9HO1EhFnrPCCuqHT2T
**Note:** New comment added should be displayed without the need for scrolling.

**Comment by NicestRudeGuy** [comment]
Understood this **issue.**  Working on the fix

**Comment by NicestRudeGuy** [comment]
@pooja-muchandikar so i found that we can have two ways to fix it .

1.  Auto scroll to the latest comment when a new comments pops up. 
2.  Have the comment section always show the lastest comments.

**Comment by NicestRudeGuy** [comment]
![](https://user-images.githubusercontent.com/9863419/99936261-bb883480-2d88-11eb-80d1-4f58bc3c433c.gif)

**Comment by NicestRudeGuy** [comment]
This works well but somewhat breaks when we have comments paginated. Trying to fix it.
![comment scroll](https://user-images.githubusercontent.com/9863419/99936370-f9855880-2d88-11eb-95b7-b5f98bbd50ed.gif)

**Comment by pooja-muchandikar** [comment]
@NicestRudeGuy 

> Auto scroll to the latest comment when a new comments pops up.

This solution sounds good, as when a latest comment is added comments section will get auto scroll and show the latest comment.

---

> Have the comment section always show the latest comments.

Do you mean hide all previous comments and give option to view more something like that?

**Comment by NicestRudeGuy** [comment]
> @NicestRudeGuy
> 
> > Auto scroll to the latest comment when a new comments pops up.
> 
> This solution sounds good, as when a latest comment is added comments section will get auto scroll and show the latest comment.
> 
> > Have the comment section always show the latest comments.
> 
> Do you mean hide all previous comments and give option to view more something like that?

Yes its always showing latest comments so everything is fine now.

**Comment by pooja-muchandikar** [comment]
Alright, if PR is ready please get it reviewed

**Comment by pooja-muchandikar** [comment]
Tested the fix with branch `GH-1464` and the fix works as expected.

```
Test Environment
WordPress: v5.6
BuddyPress: v7.1.0
rtMedia: v4.6.5 GH-1464 branch
```

**Comment by pooja-muchandikar** [comment]
Tested with `pre-release` branch and the changes looks fine.

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the changes looks fine.

**Comment by pooja-muchandikar** [comment]
Released with `v4.6.6`
